composer.json

```
@@ -15,10 +15,10 @@
         }
     ],
     "require": {
-        "php": "^7.3 || ^8.0",
+        "php": "^7.4 || ^8.0",
         "ext-pdo": "*",
         "ext-dom": "*",
-        "doctrine/dbal": "^2.11 || ^3.0",
+        "doctrine/dbal": "^3.1",
```

```suggestion
        "doctrine/dbal": "^2.13 || ^3.1",
```

Would be good to support also dbal 2 as doctrine/orm still requires dbal 2